.eslintignore

```diff
@@ -0,0 +1,3 @@
+# Workaround for .eslint error with unexpected token ...
```

Comment:
Once you remove indeterminate checbox from `table.js`, there is no longer a need for this workaround. Go ahead and delete this file and fix all es-lint errors that are prompted after you delete this file.